src/Parallel/MetavariablesTag.hpp

```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
```

Comment:
To follow our recent pattern I suggest you place this in `Parallel/Tags/Metavariables.hpp`.